daisy_workflows/build-publish/sqlserver/sql-2012-enterprise-windows-2012-r2-dc.wf.json

```diff
@@ -0,0 +1,76 @@
+{
+  "Name": "sql-2012-enterprise-windows-2012-r2-dc",
+  "Project": "gce-image-builder",
+  "Zone": "us-central1-b",
+  "GCSPath": "gs://gce-image-build-bucket/daisy/${USERNAME}",
+  "Vars": {
+    "build_date": {
+      "Value": "${TIMESTAMP}",
+      "Description": "Build datestamp used to version the image."
+    },
+    "gcs_url": {
+      "Required": true,
+      "Description": "The GCS url that the image raw file exported to."
+    },
+    "gcs_internal_url": {
```

Comment:
the internal images are only for windows. We don't need that for SQL and Containers.